api/v1beta1/disruption_types.go

```diff
@@ -186,6 +184,10 @@ func (s *DisruptionSpec) validateGlobalDisruptionScope() error {
 		}
 	}
 
+	if s.GRPC != nil && s.Level != chaostypes.DisruptionLevelPod && s.Level != chaostypes.DisruptionLevelUnspecified {
```

Comment:
```suggestion
	if s.GRPC != nil && s.Level != chaostypes.DisruptionLevelPod {
```

Reply:
@ptnapoleon I put this back after `e2e-test` failed because our `e2e-test` does not specify a level for the specs, I think unspecified should default to pod level like what I believe `init-container` is doing. We could also break up the end-to-end test file, but it seems excessive to me?